options.go

```diff
@@ -174,6 +182,9 @@ func WithMouseAllMotion() ProgramOption {
 // application, or to provide an additional non-TUI mode to your Bubble Tea
 // programs. For example, your program could behave like a daemon if output is
 // not a TTY.
+//
+// Deprecated: This option is deprecated and will be removed in a future
```

Comment:
Good point. Initially, I was like, if people don't want the TUI then they shouldn't use Bubble Tea in the application. I.e. use something like `fmt.Printf`s instead of Bubble Tea program. I don't know what're the benefits of using this option compared to something like `fmt.Printf`. Regardless, we could expose `nilRenderer` and use that in `WithRenderer(NilRenderer{})`

Reply:
i think it is to reuse the event loop without printing anything... which might make sense in some cases

Reply:
I will export the `NilRenderer` type and update the godocs.